pages/common/git-whatchanged.md

```diff
@@ -0,0 +1,17 @@
+# git whatchanged
+
+> Show what has changed with recent commits or files.
+> See also `git log`.
+> More information: <https://git-scm.com/docs/git-whatchanged>.
+
+- Display log of recent commits:
```

Comment:
```suggestion
- Display commit logs and diff outputs for recent commits:
```

Reply:
well thats just the man page, I thought tldr aimed to shorten and simplify man pages, not copying content from them